### PR TITLE
Improve compatibility with react-widgets

### LINKED ISSUE #152: Calendar dropdown styles are busted
pasted_image_at_2015_01_27_10_59_am

### PR DESCRIPTION
**Pull request by grapegravity**
Fixes a number of style issues introduced after we upgraded to React 0.12, and bumps react-widgets to 2.2.1.
Also improves isolation of component styles; previously the calendar dropdown's styles affected the tag-input.
Fixes #152.

### CODE REVIEW

**Comment by nelix** [comment]
:+1:

File: packages/calendar-dropdown/calendar-dropdown.scss [hunk 1]
```diff
@@ -43,123 +43,122 @@ $state-color-hover: #333;
...
+  padding-right: 1.9em;
 
-  .rw-input {
-    border-top-left-radius: $input-border-radius;
-    border-bottom-left-radius: $input-border-radius;
-  }
+  &.rw-widget {
+    position: relative;
+    outline: 0;
+    -moz-background-clip: border-box;
```
Comment (ojame) [inline packages/calendar-dropdown/calendar-dropdown.scss]: autoprefixer

Comment (grapegravity) [inline packages/calendar-dropdown/calendar-dropdown.scss]: This hasn’t actually changed any of the styles, just the order of the selectors. Happy to trawl through and get rid of them though if that’s what you want!

**Comment by ojame** [comment]
image

This is 'fixed' in `stack-client` by: https://github.com/macropodhq/stack-client/commit/eaa14bca6eb4ca11f40457113f516983d312f518

---

Apart from that, this is legit. I will merge when thing is fixed.

**Comment by grapegravity** [comment]
4199bcd should fix that for good :sweat_smile:

**Pull request merged**